lsp-sample/README.md

```diff
@@ -28,11 +28,11 @@ It also includes an End-to-End test.
 
 - Run `npm install` in this folder. This installs all necessary npm modules in both the client and server folder
 - Open VS Code on this folder.
-- Press Ctrl+Shift+B to compile the client and server.
-- Switch to the Debug viewlet.
-- Select `Launch Client` from the drop down.
-- Run the launch config.
-- If you want to debug the server as well use the launch configuration `Attach to Server`
-- In the [Extension Development Host] instance of VSCode, open a document in 'plain text' language mode.
+- Press Ctrl+Shift+B to start compiling the client and server in [watch mode](https://code.visualstudio.com/docs/editor/tasks#:~:text=The%20first%20entry%20executes,the%20HelloWorld.js%20file.).
+- Switch to the Run and Debug panel in the Sidebar (Ctrl+Shift+D).
```

Comment:
Calling it the "view" here is still right 🙂

Reply:
Do you mean calling Run and Debug section "Debug viewlet"? Or calling it "View"? I wasn't sure what the approved terminology is there, but "viewlet" seemed unfamiliar. I am happy with "View".
![The Run and Debug div in red dashes](https://user-images.githubusercontent.com/5055400/150184042-1748ce17-6c7d-4862-acb5-d52294e7ec88.png)

Reply:
"View" is good here.